content/en/tracing/trace_explorer/trace_view.md

```diff
@@ -167,6 +167,15 @@ Click **View in ASM** to investigate further using [Datadog Application Security
 {{< img src="tracing/trace_view/security_tab.png" alt="Security tab" style="width:90%;">}}
 
 [1]: /security/application_security/how-appsec-works/
+{{% /tab %}}
+{{% tab "Code Hotspots" %}}
+
+View [Code Hotspots][1] to identify lines of code related to performance issues. The values on the left side represent the time spent in each method call during the selected span.
+
+{{< img src="profiler/code_hotspots_tab.png" alt="Code Hotspots tab showing time spent in each method for a selected span" style="width:90%;">}}
+
+[1]: /profiler/connect_traces_and_profiles/
```

Comment:
```suggestion
[1]: /profiler/connect_traces_and_profiles/#identify-code-hotspots-in-slow-traces
```
Let's link right to the section header, since the entire page covers more than just code hotspots.